larray/core.py

```diff
@@ -6237,21 +6237,24 @@ def method(self, *args, **kwargs):
             return self._aggregate(func, args, kwargs,
                                    keepaxes=keepaxes,
                                    commutative=commutative)
+        def method_by(self, axis, **kwargs):
```

Comment:
wouldn't it be better to accept *args so that you can use several axes in *_by?
eg. sum_by(age, sex)